Mathlib/Topology/UniformSpace/UniformEmbedding.lean

```diff
@@ -265,6 +265,21 @@ theorem isComplete_image_iff {m : α → β} {s : Set α} (hm : UniformInducing
   simp_rw [IsComplete, imp.swap (a := Cauchy _), ← mem_Iic (b := 𝓟 _), fact1.forall fact2,
     hm.cauchy_map_iff, exists_mem_image, map_le_iff_le_comap, hm.inducing.nhds_eq_comap]
 
+/-- If `f : X → Y` is an `Inducing` map, the image `f '' s` of a set `s` is complete
+  if and only if `s` is complete. -/
```

Comment:
You say `Inducing`, should be `UniformInducing`. Also, LHS/RHS disagree with the lemma (see also below).